Samples~/SampleGame/Assets/Scripts/ClientBootstrap.cs

```diff
@@ -67,12 +67,6 @@ private void ShowSignIn()
             _signInScreen.Hide();
             ShowSignUp();
         });
-
-        if (_gameLiftSettings.IsLocalTest)
-        {
-            _signInScreen.SetHint("You are currently running in Local mode. You may enter any email/password to successfully login");
```

Comment:
Isn't this still the case when in anywhere mode? We return mocked responses for all UserIdentity calls from the GameLiftCoreApi class